packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -779,7 +768,8 @@ export const setDeviceMetadataKey =
         const { device } = getState().suite;
         if (!device || !device.state || !device.connected) return;
 
-        if (device.metadata.status === 'enabled') return;
+        // in case of migration device.metadata.status is already enabled
+        // if (device.metadata.status === 'enabled') return;
```

Comment:
what about to delete it instead of just commenting out?

Reply:
should be removed now